### ISSUE TITLE
Bug: --plurals command option doesn't seem to be doing anything

### ISSUE THREAD

**Issue by nonetallt** [fumeapp/modeltyper#29]
My project's code output does not change when running `php artisan model:typer --plurals` compared to `php artisan model:typer`. In comparison `php artisan model:typer --all` works as expected.

Admittedly, this might be due to the changes on my local version but I don't think that's the case. Can someone confirm that `--plurals` works as intended?

It seems like the problematic line is this one: https://github.com/fumeapp/modeltyper/blob/master/src/Commands/ModelTyperCommand.php#L61. 

```php
$plurals = $this->option('json') || $this->option('all');
```
Was probably supposed to be:

```php
$plurals = $this->option('plurals') || $this->option('all');
```

**Comment by acidjazz** [comment]
@nonetallt I can confirm that --plurals does nothing but --all works fine

**Comment by acidjazz** [comment]
confirmed fixed in v2.0.1
